### PR TITLE
Cookbook - CLM

### PR DESCRIPTION
**Pull request by Loquacity**
# Description

Adds a CLM recipe to the cookbook

# Target branches

Which documentation version does this PR apply to?

- [x] Master (Default)
- [ ] Manager-4.1
- [ ] Manager-4.0

# Links

- Partial https://github.com/SUSE/spacewalk/issues/14304
- Epic https://github.com/SUSE/spacewalk/issues/12021

### CODE REVIEW

File: modules/cookbook/nav-cookbook.adoc [hunk 1]
```diff
@@ -10,8 +10,8 @@ endif::[]
 
 
 * xref:cookbook-overview.adoc[Cookbook]
-** xref:recipe1.adoc[Recipe 1]
-** xref:receipe2.adoc[Recipe 2]
+** xref:recipe-onboarding.adoc[Recipe - Onboarding]
```
Comment (keichwa) [inline modules/cookbook/nav-cookbook.adoc:13]: I think short dashes in titles are a little bit Unamerican.  Maybe, we'd better use colons?

File: modules/cookbook/pages/recipe-clm.adoc [hunk 2]
```diff
@@ -0,0 +1,81 @@
+[[cookbook-clm]]
+= Content Lifecycle Management
+
+If you are managing a lot of clients and you need to apply customized packages to them, you can use CLM to manage your packages.
```
Comment (keichwa) [inline modules/cookbook/pages/recipe-clm.adoc:4]:
```suggestion
If you are managing a lot of clients and you need to apply customized packages to them, you can use content lifecycle management (CLM) to manage your packages.
```

File: modules/cookbook/pages/recipe-clm.adoc [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+Content lifecycle management (CLM) allows you to customize and test packages before updating production clients.
```
Comment (keichwa) [inline modules/cookbook/pages/recipe-clm.adoc:5]:
```suggestion
CLM allows you to customize and test packages before updating production clients.
```

File: modules/cookbook/pages/recipe-clm.adoc [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+It is also useful if you need to apply updates during a limited maintenance window.
+
+
+
+== Problem
+
+Content lifecycle management allows you to select software channels as sources, adjust them as required for your environment, and thoroughly test them before installing onto your production clients.
```
Comment (keichwa) [inline modules/cookbook/pages/recipe-clm.adoc:12]: And now the question is whether to use the abbrev always or not always ;)

**Review by keichwa — APPROVED** [review]
I'd avoid some "your" words (such as "your production clients"), but I know that people often think different.

File: modules/cookbook/nav-cookbook.adoc (hunk 1 above)
Comment (Loquacity) [reply modules/cookbook/nav-cookbook.adoc:13]: I don't even know if we need to prefix each with 'recipe' actually, considering that's all that's really in this book.

File: modules/cookbook/pages/recipe-clm.adoc (hunk 4 above)
Comment (Loquacity) [reply modules/cookbook/pages/recipe-clm.adoc:12]: Spell it out the first time in each section, than abbrev after that?

Comment (keichwa) [reply modules/cookbook/pages/recipe-clm.adoc:12]: sounds like a plan :)

**Review by keichwa — APPROVED** [review]

**Comment by Loquacity** [comment]
> I'd avoid some "your" words (such as "your production clients"), but I know that people often think different.

We have a tendency to use this in our docs generally (although not too much), but I think in this guide especially, it's OK. The intention of this guide was always to be slightly less stiff than the rest of our docs, anyway. That's also something we know from surveys that our readers want.

**Pull request merged**